syzLLM_server.py

```
@@ -116,58 +153,70 @@ class SamplingMethod(Enum):
     #    return beam_search_one_step_with_diversity(logits, beam_width, diversity_penalty)
     elif sampling_method == SamplingMethod.TOP_K:
-        return top_k(logits)
+        return top_k(logits, n)
     else:
         raise ValueError("Invalid sampling method specified.")
 
 
-def sample_with_temperature(logits, temperature=1.0):
+def indices_to_tokens(indices):
+    syscalls = []
+    for indices in indices:
```

Ruff

Loop control variable `indices` overrides iterable it iterates

(B020)